### DISCUSSION TITLE
Zigbee contatto pulito con COM

### DISCUSSION THREAD

**Discussion by amassa3** [indomus/forum#11] [Hardware & Firmware]
Buongiorno a tutti, mi sto avvicinando alla domotica ed il mio obiettivo sarebbe gestire tutto sotto protocollo Zigbee.
Principalmente avrei i termostati (indipendenti per ogni camera, con valvole su ogni settore del riscaldamento a pavimento), le tapparelle e la VMC.

Sembra assurdo, ma non trovo alcuno switch (anche modificandolo!) che mi permetta di gestire un semplice COM / NO / NC (per gestire la velocità della ventola della VMC).
Chi mi aiuta?
![IMG_20210321_163107017](https://user-images.githubusercontent.com/81216005/112228267-560e5c80-8c31-11eb-9799-b5bb06e364d6.jpg)

**Comment by KC-inDomus** [comment]
teoricamente lo potresti fare con un sonoff basic zigbee modificato a contatto pulito

**Reply by amassa3** [reply]
Mi mancherebbe il NC (o NO, a seconda di come lo imposto) se non sbaglio.
O mi sbaglio? Hai per caso uno schema?

**Reply by KC-inDomus** [reply]
no, non ho lo schema ma in rete dovrebbe essere disponibile. NC/NO li regoli su base logica, non è un problema

**Reply by amassa3** [reply]
Io li ho bisogno entrambi, più un comune.
Praticamente avrei bisogno un So off R4 completo, ma solo un canale.
Ora cerco uno schema, grazie!

**Reply by gio-dot** [reply]
La via più semplice è usare un Sonoff ZB mini e con la sua uscita comandare un relè a 220V tipo Finder il quale ha C-NC-NO. Se vuoi ti disegno lo schema

**Reply by gio-dot** [reply]
![Screenshot_20210324-080833_Chrome](https://user-images.githubusercontent.com/55327160/112269125-3d756500-8c78-11eb-8737-fe4c027214be.jpg)

**Reply by gio-dot** [reply]
![Screenshot_20210324-080802_Chrome](https://user-images.githubusercontent.com/55327160/112269147-48c89080-8c78-11eb-8706-e68e54b33ab7.jpg)

**Reply by gio-dot** [reply]
Relè + zoccolo. Li trovi nei negozi di materiale elettrico /elettronico

**Reply by amassa3** [reply]
Grazie!
Un 26.03 dovrebbe fare al caso mio, visto che commuta le due uscite tra 1 NO ed 1 NC

**Reply by pinnadaniele** [reply]
@gio-dot io dovrei fare lo stesso però comandando una valvola C-NO-NC che ha bisogno della 220v, c'è un Finder che gestisce in uscita la 220V? se si potrei avere gentilmente lo schema di collegamento di ZBMINIR2 e Finder?
